cwms_radar_api/src/main/java/cwms/radar/formatters/JsonV1.java

```diff
@@ -4,37 +4,137 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import cwms.radar.data.dto.CwmsDTO;
 import cwms.radar.data.dto.Location;
+import cwms.radar.data.dto.LocationCategory;
+import cwms.radar.data.dto.LocationGroup;
 import cwms.radar.data.dto.Office;
 import io.javalin.http.BadRequestResponse;
-import io.javalin.plugin.json.JavalinJson;
+import io.javalin.plugin.json.JavalinJackson;
 import service.annotations.FormatService;
 
-@FormatService(contentType = Formats.JSON, dataTypes = {Office.class,Location.class})
+@FormatService(contentType = Formats.JSON, dataTypes = {Office.class, Location.class})
 public class JsonV1 implements OutputFormatter{
 
+	private final ObjectMapper om;
+
+	public JsonV1()
+	{
+		this(JavalinJackson.getObjectMapper());
+	}
+
+	public JsonV1(ObjectMapper om)
+	{
+		this.om = om.copy();
+		this.om.setPropertyNamingStrategy(PropertyNamingStrategy.KEBAB_CASE);
+		this.om.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+	}
+
 	@Override
 	public String getContentType() {
 		return Formats.JSON;
 	}
 
 	@Override
-	public String format(CwmsDTO dto) {
-		OfficeFormatV1 fmtv1 = new OfficeFormatV1(Arrays.asList((Office)dto) );		
-		return JavalinJson.toJson(fmtv1);
+	public String format(CwmsDTO dto)
+	{
+		Object fmtv1 = buildFormatting(dto);
+		try
+		{
+			return om.writeValueAsString(fmtv1);
+		}
+		catch(JsonProcessingException e)
+		{
+			throw new FormattingException("Could not format:" + dto, e);
+		}
 	}
 
 	@Override
-	public String format(List<? extends CwmsDTO> daoList) {
-        if( daoList.get(0) instanceof Office ){
-            OfficeFormatV1 fmtv1 = new OfficeFormatV1(
-                daoList.stream().map( office -> (Office)office ).collect(Collectors.toList())
-            );
-            return JavalinJson.toJson(fmtv1);
-        } else {
-			throw new BadRequestResponse(String.format("Format {} not implemented for this data type"));
+	public String format(List<? extends CwmsDTO> daoList)
+	{
+		Object wrapped = buildFormatting(daoList);
```

Comment:
I like this change, the whole things flows better when reading it. Also a good idea not to overwrite the Javalin root level ObjectMapper since we may have other format choices in the future.